v13.3.0 incompatible with react-native <0.68.0

# Bug
Android build fails due to an unknown symbol:

```
./node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgViewManager.java:177: error: cannot find symbol
        method.invoke(view, PointerEvents.parsePointerEvents(pointerEventsStr));
                                         ^
  symbol:   method parsePointerEvents(String)
  location: class PointerEvents
```

The problem is [SvgViewManager.java](https://github.com/react-native-svg/react-native-svg/blob/f49728d783c952a97c29a8bdb3b4ad688fca2330/android/src/main/java/com/horcrux/svg/SvgViewManager.java#L177) references `com.facebook.react.uimanager.PointerEvents.parsePointerEvents`, which, looking at the tags for the [commit](https://github.com/facebook/react-native/commit/48f6967ae88100110160e1faf03e6c0d37e404bd) in which the method was introduced, is only available in React Native v0.68.0 and later.

We are currently on the latest version of react-native 0.67, and we will just revert to react-native-svg v13.2.0 until we can upgrade our project to the latest react-native. But I thought I'd let the maintainers know.

--- a/android/src/main/java/com/horcrux/svg/SvgViewManager.java
+++ b/android/src/main/java/com/horcrux/svg/SvgViewManager.java
@@ -21,6 +21,7 @@ import com.facebook.react.views.view.ReactViewGroup;
 import com.facebook.react.views.view.ReactViewManager;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Locale;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
@@ -174,7 +175,8 @@ class SvgViewManager extends ReactViewManager implements RNSVGSvgViewManagerInte
       if (superclass != null) {
         Method method = superclass.getDeclaredMethod("setPointerEvents", PointerEvents.class);
         method.setAccessible(true);
-        method.invoke(view, PointerEvents.parsePointerEvents(pointerEventsStr));
+        method.invoke(
+            view, PointerEvents.valueOf(pointerEventsStr.toUpperCase(Locale.US).replace("-", "_")));
       }
     } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
       e.printStackTrace();

